index.js

```diff
@@ -97,3 +101,4 @@
 
 //. [Fluture]:    https://github.com/fluture-js/Fluture
 //. [Sanctuary]:  https://sanctuary.js.org/
+//. [Unknown]:    https://github.com/sanctuary-js/sanctuary-def#Unknown
```

Comment:
Could we use `v:sanctuary-js/sanctuary-def#Unknown` here?

Reply:
Darn it. I knew I shouldn't have merged so eagerly. I'm very unfamiliar with the versioned links syntax. I'll open another PR.